app/main.py

```diff
@@ -1 +1,47 @@
-#  write your code here
+from __future__ import annotations
+
+
+class Animal:
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> str:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            self.is_hungry = False
+            print(f"Eating {self.appetite} food points...")
+            return self.appetite
+        if not self.is_hungry:
```

Comment:
It's same as `else`. But in this case you don't even need `else`, because you have a return statement in `if`